### ISSUE TITLE
NVDA Should Announce "Clickable" or something similar for Clickable, nonlink items

### ISSUE THREAD

**Issue by nvdaTrac** [nvaccess/test_nvdaTracImport#171]
**Reported by iris on 17 May 2008 23:35 UTC**
In order to differentiate from same-page links, blocks of text and mouse-overs, NVDA should announce "X Y or Z clickable".

**Comment by nvdaTrac** [comment]
**Comment by jteh on 19 May 2008 23:48 UTC**
See also #37.

Reporting of clickable elements was implemented in r2051. However, it seems that there is excessive/incorrect reporting of "clickable" on some pages. This is easily seen on MSDN pages such as:
http://msdn.microsoft.com/en-us/library/y471khhc(VS.80).aspx
I suspect this relates to there being an onClick action for some elements near the top of the hierarchy.

**Comment by nvdaTrac** [comment]
**Comment by jteh on 30 May 2008 01:15 UTC**
Replying to [jteh](comment:1):
> However, it seems that there is excessive/incorrect reporting of "clickable" on some pages.
This was improved somewhat in r2055. Clickables will no longer be announced if they have already been announced at a higher level in the hierarchy.
